tests/functional/test_uninstall.py

```diff
@@ -166,16 +169,20 @@ def test_uninstall_entry_point(script):
         setup(
             name='ep-install',
             version='0.1',
-            entry_points={"pip_test.ep":
-                          ["ep:name1 = distutils_install",
-                           "ep:name2 = distutils_install"]
-                          }
+            entry_points={{"console_scripts": ["{0}", ],
+                           "pip_test.ep":
+                           ["ep:name1 = distutils_install",
+                            "ep:name2 = distutils_install"]
+                          }}
         )
-    """))
+    """.format(console_scripts)))
+    script_name = script.bin_path.join(console_scripts.split('=')[0].strip())
     result = script.pip('install', pkg_path)
+    assert script_name.exists
```

Comment:
It's a property:

    (Pdb) p type(script_name)
    <class 'tests.lib.path.Path'>
    (Pdb) p script_name.exists
    True